css-two-column-layout/styles.css

```diff
@@ -0,0 +1,75 @@
+.container {
+  font-family: Karla, sans-serif;
+  flex-wrap: wrap;
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one {
+  flex-basis: 100%;
+  text-align: center;
+  max-width: 800px;
+  margin: 50px;
+}
+
+.column-two {
```

Comment:
I would rename the class `.column-two`. Something like `.column-one-half`,  `.column-half`, '.col-half`.